bower.json

```diff
@@ -22,7 +22,7 @@
     ],
     "dependencies": {
         "underscore": "~1.8.3",
-        "jquery": "~3.1.0",
+        "jquery": "~3.1.1",
```

Comment:
I did not know that (or, really, anything else about semantic versioning). Just read through the link, thank you for pointing it out :)